corehq/apps/ivr/tests/util.py

```diff
@@ -61,5 +71,7 @@ def test_log_call(self):
 
         call = Call.by_domain(self.domain)[0]
         self.assertEqual(call.couch_recipient_doc_type, 'CommCareCase')
-        self.assertEqual(call.couch_recipient, self.case.get_id)
+        self.assertEqual(call.couch_recipient, self.case.case_id)
         self.assertEqual(call.direction, INCOMING)
+
+        self.delete_case()
```

Comment:
Would be better to register this as a cleanup function to guarantee that it's called:
```
self.addCleanup(self.delete_case)
```